traffic_ops/app/lib/API/User.pm

```diff
@@ -277,6 +285,7 @@ sub current {
 			@data, {
 				"id"              => "0",
 				"username"        => $current_username,
+				"tenantId"	  => $self->current_user_tenant(),
```

Comment:
i can get this PR merged but this is the last thing that needs to change. this code will not work for ldap:

"tenantId"	  => $self->current_user_tenant(),

because there is no "current user" when you login as ldap. so we need to figure out what to do with tenantId in this scenario.